src/OpenStudy/study_openinterface.jl

```diff
@@ -374,6 +374,15 @@ function load_study(
     return data
 end
 
+function initialize_study(args...; kws...)
+    @warn """
+        `initialize_study` is deprecated and will be removed in a future release.
+        Use `load_study` instead.
+    """
+
+    return load_study(args...; kws...)
+end
```

Comment:
There is a macro for deprecations too
```julia
help?> @deprecate
  @deprecate old new [export_old=true]

  Deprecate method old and specify the replacement call new. Prevent @deprecate from exporting old by setting
  export_old to false. @deprecate defines a new method with the same signature as old.

  │ Julia 1.5
  │
  │  As of Julia 1.5, functions defined by @deprecate do not print warning when julia is run without the
  │  --depwarn=yes flag set, as the default value of --depwarn option is no. The warnings are printed from
  │  tests run by Pkg.test().

  Examples
  ≡≡≡≡≡≡≡≡≡≡

  julia> @deprecate old(x) new(x)
  old (generic function with 1 method)

  julia> @deprecate old(x) new(x) false
  old (generic function with 1 method)
```

Reply:
No, I have not. The idea is to deprecate this and adopt the new interface in the new package asap.
Deprecate works fine, but I'm afraid that users will not turn `--depwarn` on and will stick to the old interface.